packages/web/src/index.ts

```diff
@@ -402,6 +402,9 @@ export const SplunkRum: SplunkOtelWebType = {
     }).then(resp => {
       provider.resource.attributes['country'] = resp.headers.get("Cloudfront-Viewer-Country-Name") || undefined
       provider.resource.attributes['city'] = resp.headers.get("CloudFront-Viewer-City") || undefined
+      const clientAddress = resp.headers.get("CloudFront-Viewer-Address")?.split(':')
+      provider.resource.attributes['client.ip'] = clientAddress?.[0]
+      provider.resource.attributes['client.port'] = clientAddress?.[1]
```

Comment:
I don't think we need to split up. Just save client address as is. May be just whole viewer address as is? Sometimes if there is proxy it'll have multiple address

Reply:
From the [doc](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/adding-cloudfront-headers.html#cloudfront-headers-viewer-location) it seems it is a single address.